pkg/controller/storagecluster/quickstarts.go

```diff
@@ -0,0 +1,37 @@
+package storagecluster
+
+import (
+	"context"
+	"github.com/ghodss/yaml"
+	"github.com/go-logr/logr"
+	consolev1 "github.com/openshift/api/console/v1"
+	ocsv1 "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func (r *ReconcileStorageCluster) ensureQuickStarts(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
+	for _, qs := range AllQuickStarts {
+		cqs := consolev1.ConsoleQuickStart{}
+		err := yaml.Unmarshal([]byte(qs), &cqs)
+		if err != nil {
+			reqLogger.Info("failed to unmarshal consolequickstarts")
+			return err
+		}
+		found := consolev1.ConsoleQuickStart{}
+		err = r.client.Get(context.TODO(), types.NamespacedName{Name: cqs.Name, Namespace: cqs.Namespace}, &found)
+		if err != nil {
+			if errors.IsNotFound(err) {
+				reqLogger.Info("Creating Quickstarts")
+				return r.client.Create(context.TODO(), &cqs)
+			}
+		}
+		reqLogger.Info("Updating Quickstarts")
```

Comment:
Since we're inside a loop, be more specific about which Quickstart. Maybe add name and namespace?

Reply:
fixed